### ISSUE TITLE
Missing Icons in Gutter

### ISSUE THREAD

**Issue by ldelossa** [rohit-px2/nvui#152]
Hello, 

I'm trying out nvui and its been nice. 

One issue I am having is I place diagnostic glyphs in the gutter. They seem to have some trouble rendering:
![image](https://user-images.githubusercontent.com/5642902/192168506-38ef61f0-9b88-4a6d-b97b-8a60a317b709.png)

    vim.api.nvim_command('sign define DiagnosticSignError text=🄴  texthl=Error linehl= numhl=')
    vim.api.nvim_command('sign define DiagnosticSignWarn text=🅆  texthl=Warning linehl= numhl=')
    vim.api.nvim_command('sign define DiagnosticSignInfo text=🄸  texthl=Warning linehl= numhl=')
    vim.api.nvim_command('sign define DiagnosticSignHint text=🄷  texthl=Warning linehl= numhl=')

The above are the configuration lines. I believe these are just UTF characters as they are rendering right here on GitHub. Any idea?

**Comment by ldelossa** [comment]
I tried to set guifont to my normal patched font which I know exists on my system. 

I always get the error message "Could not find font for family Monospace"

**Comment by vabenil** [comment]
I think the error message "Could not find font or family Monospace" is unavoidable, it's some problem and the way nvui tries to load the default Monospace font on Linux.
Are you sure you got the syntax right? I had the same problem where nvui wasn't reading my font.

In vimscript you have to make sure there are no spaces after or before `=` and that after every space in the font you add a `\`,
like so:
```vim
set guifont=JetBrains\ Mono:h11
```
In lua  it's much easier because you just have to provide the correct string, like so:
```lua
vim.opt.guifont = "JetBrains Mono:h11"
```

**Comment by ldelossa** [comment]
Yup im sure, since neovide and goneovim work fine and can display those UTF-8 symbols with the same config i use for nvui
